src/apis/source.rs

```diff
@@ -460,17 +461,20 @@ impl SourceAPI {
             reqwest::header::AUTHORIZATION,
             reqwest::header::HeaderValue::from_str(&source_key).unwrap(),
         );
-        match client
+        let request = client
             .get(format!(
                 "{}/api/v1/api-keys/{}/auth",
                 source_api_url, access_key_id
             ))
             .headers(headers)
-            .send()
-            .await
-        {
+            .build()
+            .unwrap();
+        // Log the request details
+        debug!("request for api-keys: {:?}", request);
```

Comment:
Would this not output to the logs the sensitive `Authorization` header? That really would not be great from a security standpoint.

Reply:
Yes It does output the Authorization header and that information is logged in cloudwatch logs. But as I said, this information is logged only if the log level is set to debug or trace.